pilot/pkg/model/context.go

```diff
@@ -529,6 +546,8 @@ const (
 	// NodeMetadataIdleTimeout specifies the idle timeout for the proxy, in duration format (10s).
 	// If not set, no timeout is set.
 	NodeMetadataIdleTimeout = "IDLE_TIMEOUT"
+
+	IstioMetadataKey = "istio.io/metadata"
```

Comment:
This conflicts with `util.IstioMetadataKey` which has a different value (!!!)

Reply:
Yes, they are of different meanings. Agree we should rename differently.